### PR TITLE
Improve loqusdb integration documentation - including loqusdbapi connections -

### LINKED ISSUE #2970: Document how to connect to a loqusdbapi instance
Our current documentation contains only info on how to use loqusdb from the command line

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #2970 

Improve documentation on how to integrate loqusdb instances (executable instance or REST API call) into scout.

**How to test**:
1. Read the proposed changes in the documentation and see it makes sense.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: docs/admin-guide/loqusdb_integration.md [hunk 1]
```diff
@@ -1,71 +1,64 @@
...
+
+One or more instances of loqusdb can be initialized when the Scout server is started by editing the LOQUSDB_SETTINGS in the Scout config file.
+
+
+## Option 1: setup of a loqusdb instance executable as a binary file
 
 To avoid dependency conflicts loqusdb should be installed in an environment separate from scout.
-Install loqudb according to the instructions on the package homepage. This can be done using the same environment or another virtual environment, also the same mongodb process can be used for the loqus database however if large datasets(1000s of whole genomes) are expected it is preferable to keep loqusdb and scout in separate instances. 
+Install loqudb according to the instructions on the package homepage. This can be done using the same environment or another virtual environment, also the same mongodb process can be used for the loqus database however if large datasets(1000s of whole genomes) are expected it is preferable to keep loqusdb and scout in separate instances.
```
Comment (dnil) [inline docs/admin-guide/loqusdb_integration.md:13]: 😭 But yes.

File: docs/admin-guide/loqusdb_integration.md [hunk 2]
```diff
@@ -1,71 +1,64 @@
...
+Once a loqusdb instance is created via one one the two options (or both actually, since Scout supports the connection to multiple loqusdb instances), the `LOQUSDB_SETTINGS` parameter in the Scout config file (config.py) should be edited as shown in the following example:
 
 ```
-    LOQUSDB_SETTINGS = [
-        {"binary_path": "/path/to/bin/loqusdb", "id": "default", "config_path":"loqus.cfg"},
-        {"binary_path": "/path/to/bin/loqusdb", "id": "foo", "config_path":"bar.cfg"}
-    ]
+LOQUSDB_SETTINGS = {
+  "default" : {"binary_path": "/miniconda3/envs/loqus2.5/bin/loqusdb", "config_path": "/home/user/settings/loqus_default.yaml"}, # Example of an executable loqusdb binary file
+  "loqus_api" : {"api_url": "http://127.0.0.1:9000"}, # Example or a loqusdb instance reachable via REST API,
```
Comment (dnil) [inline docs/admin-guide/loqusdb_integration.md:35]: 💯

File: docs/admin-guide/loqusdb_integration.md [hunk 3]
```diff
@@ -1,71 +1,64 @@
...
+The example above is showing the LOQUSDB_SETTINGS parameter as a dictionary containing 2 key/values, reflecting 2 eventual instances of loqusdb, one based on the executable binary file (option 1) and one reachable via REST API (option2). To reflect the infrastructure needs, it is possible to set as many connections to loqusdb instances as required.
 
-2.1 Configure Institute via Scout Browser
-Go to sidebar 'Settings'. Find 'LoqusDB id' and enter the configured id from 
-config.py. Click Save. Your configuration is now active.
-
-![Screenshot 2020-07-10 at 12 52 16](https://user-images.githubusercontent.com/1150065/87147271-9ea50600-c2ac-11ea-9f66-333b37783d52.png)
-
-URL for institute settings: `http://localhost:5000/overview/<institute>/settings`
+Please note that the key used to define each of these instances (in this case default and loqus_api) will be later used in Scout to switch among the available instances (Scout at the moment doesn't support using different loqusdb instance at the same time, and it will be only possible to use one at the time for each institute).
```
Comment (dnil) [inline docs/admin-guide/loqusdb_integration.md:41]: Tough we will likely change that in the nearish future - as soon as cg supports balsamic loqusdb uploads I guess. It is correct as is, so your call to keep it, but we may have to remember to change it rather soon.

File: docs/admin-guide/loqusdb_integration.md [hunk 4]
```diff
@@ -1,71 +1,64 @@
...
 
+Note also that regardless of the number of loqusdb instances (key/values present in the LOQUSDB_SETTINGS file), **one dafault loqusdb instance should be present with key `default`**. This entails that if you are connecting to only one loqusdb instance, then it should be named `default`.
 
-2.2 Configure Institute via Command Line
+## Switching between loqusdb instances from the institute settings in the Scout browser
 
-    $ scout update institute  <institute> --loqusdb_id <loqusdb_id>
+This part applies only if more than one loqusdb instance is connected to Scout using the `LOQUSDB_SETTINGS` parameter present in the Scout config file. As mentioned before, **at the moment Scout support one loqusdb connection at the time for each single institute**.
```
Comment (dnil) [inline docs/admin-guide/loqusdb_integration.md:47]: Same: not sure at the moment instructions are super helpful in the long run. But sure, it is correct.

**Review by dnil — APPROVED** [review]

File: docs/admin-guide/loqusdb_integration.md (hunk 3 above)
Comment (northwestwitch) [reply docs/admin-guide/loqusdb_integration.md:41]: Yes, of course, we have an open issue about this as well, but for the time being this is the actual situation and we should document it as it is I think.

**Pull request merged**